### ISSUE TITLE
Recaptcha (friendly-captcha) in Formie Forms gone after Blitz update.

### ISSUE THREAD

**Issue by DavidKabelitz** [verbb/formie#1690]
### Describe the bug

Hi,
i am not sure who to report to. Since the last Blitz Update two days ago the friendly-captcha in all our Formie Forms has disappeared. As fallback we activated recaptcha by google which still works even with the cached pages.

Craft 4 and everything at latest versions.

Thanks

### Steps to reproduce

Blitz and friendly-captcha active, nothing else is special.

### Form settings

-

### Craft CMS version

Craft Pro 4.6.0

### Plugin version

2.1.2

**Comment by engram-design** [comment]
And just to be clear, disabling Blitz makes the captcha reappear? What about clearing the Blitz cache?

**Comment by DavidKabelitz** [comment]
Yes, without cache friendly captcha is visible...with "fresh" cache also. We were just wondering why it disappeared on all forms. We have 4 different forms on that site.

**Comment by engram-design** [comment]
So with Blitz's cache refreshed, the captchas _are_ appearing now? As for why they disappeared, I'm not quite sure. Rendering is all taken care of via JS, so unless the pages were cached before you enabled the captcha, I'm not sure.

**Comment by DavidKabelitz** [comment]
Yes still there. When it was gone the placeholder container itself was there, just without code. Maybe we can close this one and it was nothing, it's fine for 3 days now. Just a weird thing.

**Comment by DavidKabelitz** [comment]
We can close this, it's still there so maybe it was something wrong with the cache
